Instahomes/instahomes_web_app
# Set up React, Home Page and Product Page

Summary:
1. Set up CRA
2. Finish the home page

Test Plan:
1. cd to the `frontend` folder. Run `yarn` to install the packages.
2. Start the server with `yarn start`.
3. It will redirect to localhost:3000. Go through the home page to see if it looks okay.
4. Go to localhost:3000/listing. Check if the Product page looks okay

Responsiveness isn't done yet but I'll work on that later on after the Listing page.
